emission/analysis/modelling/similarity/od_similarity.py

```diff
@@ -16,6 +16,16 @@ def extract_features(self, trip: ecwc.Confirmedtrip) -> List[float]:
         return ctfe.od_features(trip)
 
     def similarity(self, a: List[float], b: List[float]) -> List[float]:
-        o_dist = ecc.calDistance([a[0], a[1]], [b[0], b[1]])
-        d_dist = ecc.calDistance([a[2], a[3]], [b[2], b[3]])
-        return [o_dist, d_dist]
\ No newline at end of file
+        """
+        a : a list of point features that can take either of two forms
+                    1. [point1_latitude,point1_longitude]  
+                    2. [point1_latitude,point1_longitude,point2_latitude,point2_longitude] 
+                    
+        b : a list of point features that can take either of two forms
+                    1. [point1_latitude,point1_longitude]  
+                    2. [point1_latitude,point1_longitude,point2_latitude,point2_longitude] 
+        """
+        point_dist = [ecc.calDistance(a[i:i+2], b[i:i+2]) 
```

Comment:
The origin/destination decision is moved upstream to `emission/analysis/modelling/trip_model/greedy_similarity_binning.py` in this commit. So we don't need separate  `calcDistance` calls.

Reply:
Yeah `calDistance` is a bit funky in that it returns two outputs. if we have more issues with this, we should consider simplifying further but I am fine with this for now.